frontend/src/utils/MadLibs.ts

```diff
@@ -252,7 +253,10 @@ if (SHOW_CANCER_SCREENINGS) {
   CATEGORIES_LIST.push({
     title: 'Cancer',
     definition: '',
-    options: CANCER_CATEGORY_DROPDOWNIDS as unknown as DropdownVarId[],
+    options: [
+      ...CDC_CANCER_CATEGORY_DROPDOWNIDS,
+      ...CANCER_CATEGORY_DROPDOWNIDS,
```

Comment:
lets rename this CANCER_SCREENING_CATEGORY_DROPDOWNIDS to avoid confusion with the new CDC_CANCER one you're adding